packages/block-library/src/post-content/edit.js

```diff
@@ -2,13 +2,28 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
 import {
 	useBlockProps,
 	__experimentalUseInnerBlocksProps as useInnerBlocksProps,
+	__experimentalUseEditorFeature as useEditorFeature,
+	store as blockEditorStore,
 } from '@wordpress/block-editor';
 import { useEntityBlockEditor } from '@wordpress/core-data';
 
-function Content( { postType, postId } ) {
+function Content( { attributes, postType, postId } ) {
+	const themeSupportsLayout = useSelect( ( select ) => {
+		const { getSettings } = select( blockEditorStore );
+		return getSettings()?.supportsLayout;
+	} );
```

Comment:
All useSelect calls need to have their dependencies defined :) 

```suggestion
	}, [] );
```